src/go/rpk/pkg/cli/cmd/redpanda/config.go

```diff
@@ -117,8 +117,22 @@ func bootstrap(fs afero.Fs) *cobra.Command {
 	c := &cobra.Command{
 		Use:   "bootstrap [--self <ip>] [--ips <ip1,ip2,...>]",
 		Short: "Initialize the configuration to bootstrap a cluster",
-		Long:  helpBootstrap,
-		Args:  cobra.ExactArgs(0),
+		Long: `Initialize the configuration to bootstrap a cluster.
+
+This command generates a redpanda.yaml configuration file to bootstrap a
+cluster. If you are modifying the configuration file further, it is recommended
+to first bootstrap and then modify. If the file already exists, this command
+will set fields as requested by flags, and this may undo some of your earlier
+edits.
+
+The --ips flag specifies seed servers (ips, ip:ports, or hostnames) that this
+broker will use to form a cluster.
+
+By default, redpanda expects your machine to have one private IP address, and
```

Comment:
```suggestion
By default, Redpanda expects your machine to have one private IP address, and
Redpanda will listen on it. If your machine has multiple private IP addresses,
you must use the --self flag to specify which IP address Redpanda should listen on.
```

Reply:
per DMs -- IIRC, our guidance currently is to use lowercase when referring to the process, and uppercase when referring to the company. We use `redpanda` pretty widely at the moment, so we'll keep the lowercase for now for consistency and for current guidance, but we can revisit wholesale in the future.

Reply:
Chatted with Travis B about these: seems the current guidance for these descriptions are to have Redpanda the company be in caps, and redpanda the binary be lowercase. I'll follow that, but perhaps there's broader changes that are worth making to make that clear (e.g. backticks everywhere when lowercase?)